Sources/Extensions/UIKit/UIWindowExtensions.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  UIWindowExtensions.swift
+//  SwifterSwift
+//
+//  Created by Omar Albeik on 6/2/18.
+//  Copyright © 2018 SwifterSwift
+//
+
+#if canImport(UIKit)
+import UIKit
+
+#if os(iOS)
+public extension UIWindow {
```

Comment:
Maybe put a marker here for "Methods"? So if someone adds stuff in the future they know where to put them